share/completions/lpadmin.fish

```diff
@@ -23,10 +23,9 @@ complete -c lpadmin -s o -d 'Sets the binary communications program to use when
 complete -c lpadmin -s o -d 'Sets the error policy to be used when the printer backend is unable to send the job to the printer. ' -xa 'printer-error-policy=abort-job printer-error-policy=retry-job printer-error-policy=retry-current-job printer-error-policy=stop-printer'
 complete -c lpadmin -s o -xa printer-is-shared=true -d 'Sets the destination to shared/published or unshared/unpublished'
 complete -c lpadmin -s o -xa printer-is-shared=false -d 'Sets the destination to shared/published or unshared/unpublished'
-complete -c lpadmin -s o -d 'Sets the IPP operation policy associated with the destination' -xa "printer-policy=(cat /etc/cups/cupsd.conf | grep \<Policy | sed 's/<Policy \(.\+\)>/\1/')"
+complete -c lpadmin -s o -d 'Sets the IPP operation policy associated with the destination' -xa "printer-policy=(string replace -r --filter '<Policy (.*)>' '$1' < /etc/cups/cupsd.conf)"
```

Comment:
Unfortunately, this needs a check for file existance/readability, or it'll print an error.

```fish
(test -r /etc/cups/cupsd.conf; and string...
```